nnoir/nnoir/functions/gemm.py

```diff
@@ -0,0 +1,18 @@
+from .function import Function
+import numpy as np
+
+
+class Gemm(Function):
+    def __init__(self, inputs, outputs, **params):
+        required_params = {'transA', 'transB', 'alpha', 'beta'}
+        optional_params = set()
+        super(Gemm, self).__init__(inputs, outputs,
+                                   params, required_params, optional_params)
+
+    def run(self, a, b, c=None):
+        a = a if self.params['transA'] == 0 else a.T
+        b = b if self.params['transB'] == 0 else b.T
```

Comment:
Why don't you make `transA` and `transB` boolean variables?

Reply:
Because I follow [ONNX spec](https://github.com/onnx/onnx/blob/master/docs/Operators.md#attributes-26).
But boolean is more reasonable.

Reply:
I will fix it.

Reply:
🙏